src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -2840,15 +2840,27 @@ public static ConversionOperatorMemberCrefSyntax ConversionOperatorMemberCref(Sy
                 type: type,
                 parameters: parameters);
         }
-<<<<<<< HEAD
 
         /// <summary>Creates a new ExtensionDeclarationSyntax instance.</summary>
-        public static ExtensionDeclarationSyntax ExtensionDeclaration(SyntaxList<AttributeListSyntax> attributeLists, SyntaxTokenList modifiers, SyntaxToken implicitOrExplicitKeyword, SyntaxToken keyword, SyntaxToken identifier, TypeParameterListSyntax? typeParameterList, ForTypeSyntax? forUnderlyingType, BaseListSyntax? baseList, SyntaxList<TypeParameterConstraintClauseSyntax> constraintClauses, SyntaxList<MemberDeclarationSyntax> members)
-            => SyntaxFactory.ExtensionDeclaration(attributeLists, modifiers, implicitOrExplicitKeyword, keyword, identifier, typeParameterList,
-                forUnderlyingType, baseList, constraintClauses,
-                SyntaxFactory.Token(SyntaxKind.OpenBraceToken), members, SyntaxFactory.Token(SyntaxKind.CloseBraceToken), semicolonToken: default);
-||||||| e9f1cd29802
-=======
+        public static ExtensionDeclarationSyntax ExtensionDeclaration(SyntaxList<AttributeListSyntax> attributeLists, SyntaxTokenList modifiers, SyntaxToken implicitOrExplicitKeyword, SyntaxToken identifier, TypeParameterListSyntax? typeParameterList, ParameterListSyntax? parameterList, ForTypeSyntax? forUnderlyingType, BaseListSyntax? baseList, SyntaxList<TypeParameterConstraintClauseSyntax> constraintClauses, SyntaxList<MemberDeclarationSyntax> members)
```

Comment:
I think it would be preferred to not make changes to the shape of this public API in a merge PR

Reply:
I took a stab at a more minimal change (new final commit after force push is https://github.com/dotnet/roslyn/commit/2b0adb74b7ccbf5d2905d283c376f6cdb4c2711f). I'm able to avoid any test changes I can't avoid public API changes (because of unavoidable `syntax.xml` change), but I can shrink it a bit (avoid optional `SyntaxFactory.cs` change).